.changeset/forty-ants-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': minor
+---
+
+Remove experimental TabPanels component in preference of UnderlinePanels
```

Comment:
As far as I understand, we don't have an internal usage for this component but I am wondering in case for open source usages should we release this as "potentially major"?

Reply:
Good point, I'm honestly not sure 🤷 It'd be great if we wouldn't even need to do a semver bump for this since it's under experimental/drafts but it's clearly nice to know if one of these things get removed. I think because of that I'm torn since we definitely don't want these to be considered under semver but it's also nice to communicate this stuff 😕 

This is where the `minor` bump came from just to say that things are changing but happy to say `major` to make this simpler 👍

Reply:
Oh this is draft!! Sorry I missed that. Yeah no worries. minor sounds good then.